### ISSUE TITLE
References to stats jsons broken

### ISSUE THREAD

**Issue by grischard** [osmbe/road-completion#399]
https://osmbe.github.io/road-completion/ loads several json files for stats in the background:

https://osmbe.github.io/road-completion/data/be-brussels.json
https://osmbe.github.io/road-completion/data/be-flanders.json
https://osmbe.github.io/road-completion/data/be-wallonia-bra.json
https://osmbe.github.io/road-completion/data/be-wallonia-hai.json
https://osmbe.github.io/road-completion/data/be-wallonia-lie.json
https://osmbe.github.io/road-completion/data/be-wallonia-lux.json
https://osmbe.github.io/road-completion/data/be-wallonia-nam.json
https://osmbe.github.io/road-completion/data/lu-luxembourg.json
https://osmbe.github.io/road-completion/data/xk-kosovo.json

All of these json files except be-brussels are 404, and the stats are not displayed.

**Comment by jbelien** [comment]
Yes, it's indeed a known issue for which I don't have a solution yet.
It's caused during build process (see https://github.com/osmbe/road-completion/actions/runs/5005785588/jobs/8970212860#step:5:12).
My guess it's that too many people try to use GitHub API from GitHub Hosted Runners and we hit a rate limit.
